templates/uwsgi_systemd.service.erb

```diff
@@ -0,0 +1,15 @@
+#
+# This file is managed by puppet class 'uwsgi'
+#
+# uWSGI Emperor process systemd service
+#
+[Unit]
+Description=uWSGI Emperor
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/sbin/uwsgi --die-on-term --ini <%= @config_file %>
```

Comment:
Should you parameterise the uwsgi binary location or just leave the path off? I'm just imagining other distros that use systemd may not install uwsgi into sbin.